nrfx_glue.h

```diff
@@ -248,9 +248,10 @@ extern const u32_t z_bt_ctlr_used_nrf_ppi_channels;
 #endif
 
 #if defined(CONFIG_PWM_NRF5_SW)
+#define PWM_NRF5_SW_DEV DT_INST(0, nordic_nrf_sw_pwm)
```

Comment:
Node identifiers are being called `FOO_NODE` instead of `FOO_DEV` elsewhere, might be nicer to keep that consistent. I think NODE is cleaner since 'dev' suggests 'device'.

Reply:
Changed to NODE.